app/src/main/java/org/oppia/app/customview/SegmentedCircularProgressView.kt

```diff
@@ -88,22 +97,36 @@ class SegmentedCircularProgressView : View {
       // Draws arc for every finished chapter.
       for (i in 0 until chaptersFinished) {
         val startAngle =
-          angleStartPoint + i * (sweepAngle + STROKE_DASH_GAP_IN_DEGREE) + STROKE_DASH_GAP_IN_DEGREE / 2
+          angleStartPoint + i * (sweepAngle + STROKE_DASH_GAP_IN_DEGREE) +
+            STROKE_DASH_GAP_IN_DEGREE / 2
         canvas.drawArc(baseRect!!, startAngle, sweepAngle, false, chapterFinishedArcPaint)
       }
       angleStartPoint += chaptersFinished * (sweepAngle + STROKE_DASH_GAP_IN_DEGREE)
       // Draws arc for every unfinished chapter.
       for (i in 0 until chaptersNotFinished) {
         val startAngle =
-          angleStartPoint + i * (sweepAngle + STROKE_DASH_GAP_IN_DEGREE) + STROKE_DASH_GAP_IN_DEGREE / 2
+          angleStartPoint + i * (sweepAngle + STROKE_DASH_GAP_IN_DEGREE) +
+            STROKE_DASH_GAP_IN_DEGREE / 2
         canvas.drawArc(baseRect!!, startAngle, sweepAngle, false, chapterNotFinishedArcPaint)
       }
     } else if (totalChapters == 1) {
       // Draws entire circle for finished an unfinished chapter.
       if (chaptersFinished == 1) {
-        canvas.drawArc(baseRect!!, angleStartPoint, 360f, false, chapterFinishedArcPaint)
+        canvas.drawArc(
+          baseRect!!,
+          angleStartPoint,
+          360f,
```

Comment:
Are we not mentioning  what are these constants?(this and the below one)

Reply:
Actually, I haven't worked on this code but I guess whoever has written this code didn't mention what are these constants because I think `drawArc()` is some in-built method and in the editor in the android studio it might show the argument in suggestion, that comes in faded grey.

Reply:
@nikita Your point is correct, we should mention what this value is. 
@anandwana Can you please update the PR so that there is an inline comment for this value.

Reply:
@rt4914 Sure

Reply:
@nikitamarysolomanpvt @rt4914 
Should it look like this?

```
canvas.drawArc(
          /* oval */ baseRect!!,
          /* startAngle */ angleStartPoint,
          /* sweepAngle */ 360f,
          /* useCenter */ false,
          /* paint */ chapterFinishedArcPaint
        )
```